### PR TITLE
More comments in libgit2 tests

### PR DESCRIPTION
**Pull request by kshyatt**
Hopefully makes it clearer what things are doing and why they're doing it that way. This is *sort of* a `doc` PR...

### CODE REVIEW

File: test/libgit2.jl [hunk 1]
```diff
@@ -685,6 +714,8 @@ mktempdir() do dir
                 blob = LibGit2.GitBlob(repo, id)
                 @test LibGit2.isbinary(blob)
                 len1 = length(blob)
+
+                # test showing a GitBlog
```
Comment (tkelman) [inline test/libgit2.jl]: GitBlob

Comment (kshyatt) [inline test/libgit2.jl]: You don't know that this isn't a `GitBlob`... of my blog posts.

File: test/libgit2.jl [hunk 2]
```diff
@@ -793,6 +835,10 @@ mktempdir() do dir
         LibGit2.set!(cfg, "user.name", "AAAA")
         LibGit2.set!(cfg, "user.email", "BBBB@BBBB.COM")
         try
+
```
Comment (tkelman) [inline test/libgit2.jl]: I'd avoid the unnecessary empty line at the start (or end) of a block

**Review by Sacha0 — APPROVED** [review]
superficially lgtm! :)

**Comment by kshyatt** [comment]
Did I forget to bump this? @tkelman good to go?

**Pull request merged**